docs/quickstarts/fluent-react-quickstart.md

```diff
@@ -0,0 +1,108 @@
+
+- The **./manifest.xml** file in the root directory of the project defines the settings and capabilities of the add-in. To learn more about the **manifest.xml** file, see [Office Add-ins XML manifest](../develop/xml-manifest-overview.md).
+- The **./src/taskpane/taskpane.html** file contains the HTML markup for the task pane and loads the Office JavaScript Library. It also tests whether the webview control supports Fluent UI React ver. 9 and displays a special message if it doesn't.
+- The **./src/taskpane/index.jsx (tsx)** file is the React root component. It loads React and Fluent UI React, ensures that the Office JavaScript library has been loaded, and applies the Fluent-defined theme. 
+- The **./src/taskpane/office-document.js (ts)** file contains the Office JavaScript API code that facilitates interaction between the task pane and the Office client application.
+- The **./src/taskpane/components/** folder contains the React component *.jss (tsx) files that create the UI.
+
+### Try it out
+
+1. Navigate to the root folder of the project.
+
+    ```command&nbsp;line
+    cd "My Office Add-in"
+    ```
+
+1. Complete the following steps to start the local web server and sideload your add-in.
+    [!INCLUDE [alert use https](../includes/alert-use-https.md)]
+
+    > [!TIP]
+    > If you're testing your add-in on Mac, run the following command before proceeding. When you run this command, the local web server starts.
+    >
+    > ```command&nbsp;line
+    > npm run dev-server
+    > ```
+
+    - To test your add-in, run the following command in the root directory of your project. This starts the local web server and opens the Office host application with your add-in loaded.
+
+        ```command&nbsp;line
+        npm start
+        ```
+
+    - To test your add-in in Office on the web, run the following command in the root directory of your project. When you run this command, the local web server starts. Replace "{url}" with the URL of a Word document on your OneDrive or a SharePoint library to which you have permissions.
+
+        [!INCLUDE [npm start:web command syntax](../includes/start-web-sideload-instructions.md)]
+
+    > [!NOTE]
+    > If this is the first time that you have sideloaded an Office add-in on your computer (or the first time in over a month), you're prompted first to delete an old certificate and then to install a new one. Agree to both prompts.
+
+1. A **WebView Stop On Load** prompt appears. Select **OK**.
+
+1. If the "My Office Add-in" task pane isn't already open, choose the **Home** tab, and then choose the **Show Taskpane** button on the ribbon to open the add-in task pane.
+
+1. Enter text into the text box and then select **Insert text**. 
+
+![Custom text inserted into the document after selecting the Insert button from the add-in task pane.](../images/word-task-pane-react-component.png)
+
+### Next steps
```

Comment:
Suggesting to remove this heading, unless we add other recommended steps, such as checking out the Fluent UI React v9 documents.